tests/ert/unit_tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -107,7 +107,12 @@ async def test_evaluator_raises_on_start_with_address_in_use(make_ee_config):
     ctx = zmq.asyncio.Context()
     socket = ctx.socket(zmq.ROUTER)
     try:
-        socket.bind(f"tcp://*:{ee_config.router_port}")
+        ee_config.router_port = socket.bind_to_random_port(
+            "tcp://*",
+            min_port=min(ee_config.port_range),
+            max_port=max(ee_config.port_range),
```

Comment:
Here I'd keep it explicit; ie. as it was:
```python
socket.bind(f"tcp://*:{ee_config.router_port}")
```

Reply:
router_port isn't set when creating EvaluatorServerConfig anymore. it used find_available_socket to set that.